tools/analysis_tools/vis_scheduler.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
We can add docs in `useful tools` to tell users how to use this, and give some results there.